Dockerfile

```diff
@@ -13,6 +17,12 @@ RUN bundle install --jobs 4 && npm install && npm install -g phantomjs-prebuilt
 
 EXPOSE 3000
 ADD . /myapp
-RUN RAILS_ENV=production bundle exec rake assets:precompile
+RUN if [ -z "$CUSTOM_ASSETS_URL" ]; then\
```

Comment:
The ADD directive can take a URL. So what you could do is ADD the location where you have your zipped files, unzip, and call `rake assets:precompile`. https://docs.docker.com/engine/reference/builder/#/add